lib/media/track/remoteaudiotrack.js

```diff
@@ -9,35 +9,24 @@ const RemoteMediaAudioTrack = mixinRemoteMediaTrack(AudioTrack);
  * A {@link RemoteAudioTrack} represents an {@link AudioTrack} published to a
  * {@link Room} by a {@link RemoteParticipant}.
  * @extends AudioTrack
- * @property {boolean} isSubscribed - Whether the {@link RemoteAudioTrack} is
- *   currently subscribed to
- * @property {Track.SID} sid - The {@link RemoteAudioTrack}'s SID
  * @emits RemoteAudioTrack#disabled
  * @emits RemoteAudioTrack#enabled
  * @emits RemoteAudioTrack#started
- * @emits RemoteAudioTrack#unsubscribed
  */
 class RemoteAudioTrack extends RemoteMediaAudioTrack {
   /**
    * Construct a {@link RemoteAudioTrack}.
    * @param {MediaTrackReceiver} mediaTrackReceiver - An audio MediaStreamTrack container
-   * @param {RemoteTrackSignaling} signaling - The {@link Track} signaling
+   * @param {boolean} isEnabled - whether the {@link RemoteAudioTrack} is enabled
```

Comment:
Cool, and I guess we can remove the `signaling` parameter because the RemoteAudioTrackPublication will control the RemoteAudioTrack?

Reply:
@markandrus Yes.